core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -133,34 +126,35 @@ private static void init(String sitePropertiesFile, boolean includeSystemPropert
     }
 
     // Now lets combine, order matters here
+    PROPERTIES.clear();
     mergeProperties(defaultProps);
     if (siteProps != null) {
       mergeProperties(siteProps);
     }
     mergeProperties(systemProps);
 
-    String masterHostname = PROPERTIES.get(PropertyKey.MASTER_HOSTNAME);
-    String masterPort = PROPERTIES.get(PropertyKey.MASTER_RPC_PORT);
-    boolean useZk = Boolean.parseBoolean(PROPERTIES.get(PropertyKey.ZOOKEEPER_ENABLED));
+    String masterHostname = PROPERTIES.get(PropertyKey.MASTER_HOSTNAME.toString());
```

Comment:
would it make sense to use the `get(...)` and `set(...)` methods in this method (instead of `PROPERTIES.get(...)` and `PROPERTIES.set(...)`) to avoid having to do toString()?